Source/Mac/NSColor+Hue.swift

```diff
@@ -4,26 +4,34 @@ import AppKit
 
 public extension NSColor {
 
-  public static func hex(string: String) -> NSColor {
-    var hex = string.hasPrefix("#")
-      ? String(string.characters.dropFirst())
-      : string
+  convenience init(hex: String) {
+    var hex = hex.hasPrefix("#")
+      ? String(hex.characters.dropFirst())
+      : hex
 
     guard hex.characters.count == 3 || hex.characters.count == 6
-      else { return NSColor.whiteColor().colorWithAlphaComponent(0.0) }
+      else {
+        self.init(white: 1.0, alpha: 0.0)
+        return
+    }
 
     if hex.characters.count == 3 {
       for (index, char) in hex.characters.enumerate() {
         hex.insert(char, atIndex: hex.startIndex.advancedBy(index * 2))
       }
     }
 
-    return NSColor(
+    self.init(
```

Comment:
This is nice. Should we move the same `hex -> r g b` parsing in `UIColor, NSColor` into a common function?